<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Database_add</class>
 <widget class="QDialog" name="Database_add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Database_add</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res_static.qrc">
    <normaloff>:/icon/source/res/photo/icon/title.png</normaloff>:/icon/source/res/photo/icon/title.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>481</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>名称添加</string>
   </property>
   <widget class="QComboBox" name="cmb_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>158</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>标签1</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>添加名称（1-2）</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="edt_name">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>261</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Microsoft YaHei UI&quot;;
color:rgb(0, 0, 0);
</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>841</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string>添加内容</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>输入你想要添加的函数|数据|其他文本</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>791</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_tag">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>481</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>标签添加</string>
   </property>
   <widget class="QComboBox" name="cmb_tag">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>158</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>标签1</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>添加数据库标签（0-3）</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="edt_lab">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>261</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Microsoft YaHei UI&quot;;
color:rgb(0, 0, 0);
</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>350</y>
     <width>101</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="btn_close">
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	font-family:Microsoft Yahei;
	color:white;
	background-color:rgb(14 , 150 , 254);
	border-radius:5px;
}

QPushButton:hover
{
	background-color:rgb(44 , 137 , 255);
}

QPushButton:pressed
{
	background-color:rgb(14 , 135 , 228);
	padding-left:3px;
	padding-top:3px;
}
</string>
      </property>
      <property name="text">
       <string>退出程序</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_back">
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	font-family:Microsoft Yahei;
	color:white;
	background-color:rgb(14 , 150 , 254);
	border-radius:5px;
}

QPushButton:hover
{
	background-color:rgb(44 , 137 , 255);
}

QPushButton:pressed
{
	background-color:rgb(14 , 135 , 228);
	padding-left:3px;
	padding-top:3px;
}
</string>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_ok">
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	font-family:Microsoft Yahei;
	color:white;
	background-color:rgb(14 , 150 , 254);
	border-radius:5px;
}

QPushButton:hover
{
	background-color:rgb(44 , 137 , 255);
}

QPushButton:pressed
{
	background-color:rgb(14 , 135 , 228);
	padding-left:3px;
	padding-top:3px;
}
</string>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lab_filetype_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>170</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>提示信息</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rdb_open">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>170</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>开</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>170</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>关</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res_static.qrc"/>
 </resources>
 <connections/>
</ui>
